packages/icons/src/stories/custom-mode-icon-renderer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { StyledTable } from "../icon-renderer";
+import ClassicModeIcon from "../classic-mode-icon";
+import CustomModeIcon from "./custom-mode-icon";
+
+const walkTransitWalkItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-transit-walk.json");
+
+const exampleLeg = walkTransitWalkItinerary.legs[1];
+
+const CustomModeIconRenderer = () => {
+  return (
+    <StyledTable>
+      <thead>
+        <tr>
+          <th>{exampleLeg.mode}</th>
+          <th>Result</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Classic Mode</td>
+          <td>
+            <div>
```

Comment:
Do we need those divs?

Reply:
There's `div` styling in the `StyledTable` that keeps the icons the right size!